src/test/kotlin/snyk/errorHandler/SentryErrorReporterTest.kt

```diff
@@ -53,6 +53,17 @@ class SentryErrorReporterTest {
 
         verify(exactly = 0) { Sentry.captureException(any()) }
     }
+    @Test
+    fun `captureException should not send exceptions to Sentry when URL is not a permitted environment`() {
```

Comment:
What's the difference to the test above? Could the tests maybe be combined like you did above?

Reply:
I used yo have one test that validate to 0 and one that validate to 1. In last review, I changed so both validated to 0. I now reverted back, so that one validate to 0 and one validate to 1.

Reply:
Ill update it, so we have one that evaluate to 0 and one that evaluate to 1.